scripts/turtles-dev.sh

```diff
@@ -54,6 +55,9 @@ helm upgrade ngrok ngrok/kubernetes-ingress-controller \
 
 kubectl apply -f test/e2e/data/rancher/ingress-class-patch.yaml
 
+docker pull $RANCHER_IMAGE
+kind load docker-image rancher/rancher:$RANCHER_VERSION --name $CLUSTER_NAME
```

Comment:
Nitpick: I would move it to right after the `kind create cluster` creation command (line 34) to pull and load it.

Reply:
Makes sense. Done.